kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/MessageItem.tsx

```diff
@@ -12,20 +12,23 @@ export interface MessageItemProp {
   offset: TopicMessage['offset'];
   timestamp: TopicMessage['timestamp'];
   content?: TopicMessage['content'];
+  keyMessage?: TopicMessage['key'];
 }
 
 const MessageItem: React.FC<MessageItemProp> = ({
   partition,
   offset,
   timestamp,
   content,
+  keyMessage,
 }) => {
   const { copyToClipboard, saveFile } = useDataSaver(
     'topic-message',
     (content as Record<string, string>) || ''
   );
   return (
     <tr>
+      <td style={{ width: 100 }}>{keyMessage}</td>
```

Comment:
I'm not sure we need to set width for key column

Reply:
done